challenges/objects-arrays.js

```diff
@@ -48,23 +70,32 @@ const graduates = [
   { id: 10, first_name: "Hube", university: "Universitat Rovira I Virgili Tarragona", email: "hlethbrig9@foxnews.com" },
 ];
 
-/* Request 1: Create a new array called universities that contains all the universities in the graduates array. This will be an array of strings.
-
+/* Request 1: Create a new array called universities that contains all the universities in the graduates array. This will be an array of strings
```

Comment:
Think about these lines of code again, does this solve the challenge? After creating a new and empty array?